src/tools/cephfs/cephfs-shell

```diff
@@ -731,8 +731,7 @@ class CephFSShell(Cmd):
                 return
             copy_to_local(root_src_dir, b'-')
         elif is_file_exists(args.remote_path):
-            copy_to_local(root_src_dir,
-                          root_dst_dir + b'/' + root_src_dir)
+            copy_to_local(root_src_dir, root_dst_dir)
```

Comment:
Would the tests `TestGetAndPut` need to be updated?

Reply:
I'll check it

Reply:
relevant test's path has been fixed

Reply:
Is there a test that validates if a file that's fetched (using `get`) under a subdirectory does not create the subdirs itself?

Reply:
I don't think so such test exists

Reply:
> I don't think so such test exists

Then its probably a good idea to add one, isn't it?

Reply:
I'll add it

Reply:
@vshankar Testcase added